e2e/tools/validator/src/validator/prometheus/__init__.py

```diff
@@ -123,6 +124,16 @@ def mape(actual: npt.ArrayLike, predicted: npt.ArrayLike) -> ValueOrError:
         return ValueOrError(value=0, error=str(e))
 
 
+def mae(actual: npt.ArrayLike, predicted: npt.ArrayLike) -> ValueOrError:
+    try:
+        actual, predicted = validate_arrays(actual, predicted)
+        return ValueOrError(value=np.abs(np.subtract(actual, predicted)).mean())
```

Comment:
IIUC,  https://www.statisticshowto.com/absolute-error/ 
the `abs` should be applied to `subtract`  and not to the mean 
```
|(1  + - 1 ) / 2 | != ( |1|  + |-1| ) / 2 
```
Hence 
```
def mean_absolute_error(actual, predicted):
    return np.mean(np.abs(actual - predicted))
```